src/features/search/pages/SearchWrapper.tsx

```diff
@@ -28,6 +28,8 @@ export const SearchWrapper = ({ children }: { children: Element }) => {
   const [debouncedSearchState, setDebouncedSearchState] = useState<SearchState>(searchState)
   const { showResults, ...parameters } = debouncedSearchState
```

Comment:
```suggestion
  const { showResults, query, ...parameters } = debouncedSearchState
```

comme ça tu passe pas query à `buildSearchParameters `